### PR TITLE
Add docs for CSV_EXPORT2

### LINKED ISSUE #193: Documentation for CSV_EXPORT2
Users request more documentation on CSV_EXPORT2.

https://www.yammer.com/statoil.com/threads/868942310481920 [user khgu]

### PR DESCRIPTION
**Pull request by berland**
Closes #193

### CODE REVIEW

**Review by oyvindeide — COMMENTED** [review]
This looks good, thanks for adding documentation! Could you add a test checking that the description and examples are valid `rst`? They are, but for future use. Could be something like this: https://github.com/equinor/semeio/blob/master/tests/jobs/correlated_observations_scaling/unit/test_examples.py

Also, can you link it to the corresponding issue?

**Comment by berland** [comment]
> Also, can you link it to the corresponding issue?

Not  enough privileges to link to issues..

**Comment by oyvindeide** [comment]
Ahh, closes #214

File: semeio/workflows/csv_export2/csv_export2.py [hunk 1]
```diff
@@ -6,6 +6,35 @@
 from fmu import ensemble
 from res.job_queue import ErtScript
 
+DESCRIPTION = """
```
[the remark this answers is not in the document]
Comment (berland) [reply semeio/workflows/csv_export2/csv_export2.py:9]: I think I rather want to extend it (pending good suggestions..)

**Comment by rnyb** [comment]
MAKE_DIRECTORY <CONFIG_PATH>/ert/output/csv
  CSV_EXPORT2 <RUNPATH_FILE> <CONFIG_PATH>/ert/output/csv/<CASE>.csv monthly F* W* TCPU TIMESTEP

I would suggest to write output to the scratch disk (instead of the project disk),
in <scarcth>/<case_dir>/share/someoutputfolder/...

**Comment by rnyb** [comment]
Would it be useful to add to the description some tips on how to avoid too large csv files?

**Comment by sondreso** [comment]
> Ahh, closes #214

This needs to be in the description of the PR :relaxed:

**Comment by oyvindeide** [comment]
> > Ahh, closes #214
> 
> This needs to be in the description of the PR ☺️ (And probably point to the issue and not the PR?)

Such high requirements! :p

**Comment by berland** [comment]
> Would it be useful to add to the examples or description some tips on how to avoid too large csv files?

Thinking of vector selection or time frequency? Example tip?

**Comment by berland** [comment]
> MAKE_DIRECTORY <CONFIG_PATH>/ert/output/csv
> CSV_EXPORT2 <RUNPATH_FILE> <CONFIG_PATH>/ert/output/csv/.csv monthly F* W* TCPU TIMESTEP
> 
> I would suggest to write output to the scratch disk (instead of the project disk),
> in .../<case_dir>/share/someoutputfolder/...

Done.

**Comment by rnyb** [comment]
> > Would it be useful to add to the examples or description some tips on how to avoid too large csv files?
> 
> Thinking of vector selection or time frequency? Example tip?

Both, i see time interval examples is covered now. Though convenient the * notation may easily dump much more vectors than the user needs(?) But maybe this should be left to the user to judge...?

File: semeio/workflows/csv_export2/csv_export2.py [hunk 2]
```diff
@@ -6,6 +6,70 @@
...
+for more details on rate handling.
+
+Columns are selected by a list of strings, where wildcards characters ``?``
+(matches exactly one character) and ``*`` (matches zero or more characters) can
+be used to select multiple columns.
+
+Column count more than 1000 gives increased probability for problems downstream,
+depending on which applications are put into use. Column count depends on the
+combination of wildcards used in this workflow and the actual vectors that are
+requested in the Eclipse DATA file. A wildcard like``W*`` could be dangerous on
```
Comment (oyvindeide) [inline semeio/workflows/csv_export2/csv_export2.py:42]: `dangerous`, at least to me, implies a security risk, which is not the case here, can you rephrase it to something else? For example `A wildcard like ``W*`` could lead to very large csv files on...`

Comment (berland) [reply semeio/workflows/csv_export2/csv_export2.py]: Agree. Fixed.

**Comment by oyvindeide** [comment]
Everything looks good from my perspective, but @rnyb is probably better placed to review the contents of the docs. So when he is happy this can be merged.

**Comment by rnyb** [comment]
> Everything looks good from my perspective, but @rnyb is probably better placed to review the contents of the docs. So when he is happy this can be merged.

I think it looks good now

**Comment by berland** [comment]
NB: "weekly" will not work before komodo 2020.10 is stable.

**Comment by oyvindeide** [comment]
> NB: "weekly" will not work before komodo 2020.10 is stable.

Is it a lot of work to split that into a separate PR that we can merge when that is closer in time? Normally that should be so close that it would not be a problem, but might be delayed.

Other than that, rebase and it is good to go.

**Comment by oyvindeide** [comment]
Great, can you also create an issue or a PR straight away so we dont forget?

**Comment by berland** [comment]
Rebased.

**Review by oyvindeide — APPROVED** [review]
LGTM! Thanks for adding docs!
